cassandra-four-zero/src/main/java/org/apache/cassandra/bridge/SSTableWriterImplementation.java

```diff
@@ -80,4 +79,28 @@ public void close() throws IOException
     {
         writer.close();
     }
+
+    @VisibleForTesting
+    static CQLSSTableWriter.Builder configureBuilder(String inDirectory,
+                                                     String createStatement,
+                                                     String insertStatement,
+                                                     RowBufferMode rowBufferMode,
+                                                     int bufferSizeMB,
+                                                     IPartitioner cassPartitioner)
+    {
+        CQLSSTableWriter.Builder builder = CQLSSTableWriter.builder()
+                                                           .inDirectory(inDirectory)
+                                                           .forTable(createStatement)
+                                                           .withPartitioner(cassPartitioner)
+                                                           .using(insertStatement);
+        if (rowBufferMode == RowBufferMode.UNBUFFERED)
+        {
+            builder.sorted();
+        }
+        else if (rowBufferMode == RowBufferMode.BUFFERED)
+        {
+            builder.withBufferSizeInMB(bufferSizeMB);
```

Comment:
nit: maybe decide a valid value range for `bufferSizeMB` and validate. `CQLSSTableWriter` accepts whatever int value. 
For the upper bound, I think 1/2 of the `spark.executor.memory` is a good limit.

Reply:
I think, for now, we leave this just a configuration option... Validating it given the Spark environment and picking a reasonable upper-bound would take some experimentation (and would likely involve more than just the executor.memory setting, as there are other config settings that deal w/ memory like memory overhead).